docs/glossary.md

```diff
@@ -11,74 +6,116 @@ sidebar_position: 27
+Application Binary Interface. ABI is a document that comprehensively defines the binary system interface between applications and the operating system on which they run.
+
+[Learn more](https://en.wikipedia.org/wiki/Application_binary_interface).
+
+<a id="term-CDN"></a>
+
+## CDN
+
+Content Delivery Network. CDNs are geographically distributed networks of servers that mirror the contents of a primary source. Having multiple servers offering the same content increases performance (reduced latency, higher download speeds) and availability.
+
+[Learn more](https://en.wikipedia.org/wiki/Content_delivery_network).
 
 <a id="term-CDT"></a>
 
 ## CDT
 
-**C**ore **D**ependency **T**ree. Core Dependency Tree packages take care of the dependencies which are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform. [Learn more](maintainer/knowledge_base.md#cdt-packages).
+Core Dependency Tree. Core Dependency Tree packages take care of the dependencies that are so close to the system that they are not packaged with conda-forge. A CDT package consists of repackaged CentOS binaries from the appropriate version, either 6 or 7 depending on user choice and platform.
 
-## CLI
+[Learn more](https://conda-forge.org/docs/maintainer/knowledge_base.html#cdt-packages).
+
+<a id="term-CFEP"></a>
 
-**C**ommand **L**ine **I**nterface. A program in which commands are entered as text, one line at a time, for a computer to execute. This is done in the Miniforge Prompt in Windows, and in a terminal in macOS and Linux. `conda` is executed in a CLI. [Learn More](https://en.wikipedia.org/wiki/Command-line_interface).
+## CFEP
+
+Conda Forge Enhancement Proposal. A CFEP is a document that outlines a suggested change to how the conda-forge project operates, from a technical standpoint as well as to address social topics such as governance and expected conduct.
+
+[Learn More](https://github.com/conda-forge/cfep/blob/main/cfep-01.md/).
 
 <a id="term-CI"></a>
 
 ## CI
 
-**C**ontinuous **I**ntegration. Continuous integration is the practice of automating the integration of code changes from multiple contributors into a single software project. [Learn More](https://en.wikipedia.org/wiki/Continuous_integration).
+Continuous Integration. Continuous integration is the practice of automating the integration of code changes from multiple contributors into a single software project.
 
-<a id="term-CFEP"></a>
+[Learn More](https://en.wikipedia.org/wiki/Continuous_integration).
 
-## CFEP
+<a id="term-CLI"></a>
+
+## CLI
+
+Command Line Interface. A program in which commands are entered as text, one line at a time, for a computer to execute. This is done in the Miniforge Prompt in Windows, and in a terminal in macOS and Linux. `conda` is executed in a CLI.
```

Comment:
I would rephrase the last two sentences to make sure people understand they are concrete examples.

Reply:
Hm. To be honest, I am not thrilled about the entire explanation. What do we mean here with CLI? Imho, it can be either "the program that takes the command-line and launches the desired program", i.e. power shell, bash, etc.; or it can be the interface offered by programs such as `conda` including parameters etc.

When I say CLI, I usually mean the latter. The explanation here starts out with the former, but than kind of mangles the two; imho, the linked explanation on wikipedia refers to the second meaning.

Reply:
So I would most like to rewrite with the second meaning. Something along the lines of
```suggestion

## CLI

Command Line Interface. A textual interface of a program that allows the user to control the program with a single command line, which is entered in a command-line environment, such as the Miniforge Prompt on Windows, or a terminal on MacOS and Linux. An example for a program offering a CLI is `conda`.
```